frontend/src/components/pages/connect/Overview.tsx

```diff
@@ -36,27 +42,36 @@ class KafkaConnectOverview extends PageComponent {
 
     async refreshData(force: boolean) {
         await api.refreshConnectClusters(force);
-        if (api.connectConnectors?.isConfigured) {
-            const clusters = api.connectConnectors.clusters;
-            if (clusters?.length == 1) {
-                const cluster = clusters[0];
-                appGlobal.history.replace(`/connect-clusters/${cluster.clusterName}`);
-            }
-        }
+        // if (api.connectConnectors?.isConfigured) {
+        //     const clusters = api.connectConnectors.clusters;
+        //     if (clusters?.length == 1) {
+        //         const cluster = clusters[0];
+        //         appGlobal.history.replace(`/connect-clusters/${cluster.clusterName}`);
+        //     }
+        // }
     }
 
     render() {
         if (!api.connectConnectors) return DefaultSkeleton;
         if (api.connectConnectors.isConfigured == false) return <NotConfigured />;
-        const settings = uiSettings.kafkaConnect;
+
+        const tabs = [
+            {
+                key: 'redpandaConnect',
```

Comment:
I guess it would improve the UX if we make each of these pages persisted in the URL. e.g. like in the Access control page.